apps/frontpage/app/versions/route.ts

```diff
@@ -1,44 +1,33 @@
-import { stringify } from 'node:querystring';
```

Comment:
The left side is a direct copy/paste of https://github.com/storybookjs/frontpage/blob/main/src/functions/versions.js, for easy diffing.